app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PostsController < ApplicationController
+  def index
+    @posts = @user.posts
+  end
```

Comment:
- [ ] [Optional] Great job on your progress! However, there's a small issue that needs attention. In your code, you are using `@posts = @user.posts`, but it seems there is no variable named `@user` defined in your index method, leading to an error: `undefined method 'posts' for nil:NilClass`. 

To resolve this issue, you should ensure that the `@user` variable is properly initialized or replaced with the correct variable that holds the user whose posts you want to fetch. Review your code and make the necessary adjustments to fix this error. Keep up the good work! 🚀👍